bigquery/biglake/bigquery_create_partitioned_biglake_table/main.tf

```
@@ -109,11 +110,20 @@ resource "google_bigquery_table" "default" {
       source_uri_prefix        = "gs://${google_storage_bucket.default.name}/publish/{dt:STRING}/{hr:STRING}/{min:STRING}"
       require_partition_filter = false
     }
+    # This enables automatic metadata refresh.
+    metadata_cache_mode = "AUTOMATIC"
   }
+
+
+  # `max_staleness` is required for automatic metadata caching and must be
+  # specified as an interval literal.
+  # https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#interval_literals
```

```suggestion
  # This sets the maximum staleness of the metadata cache to 10 hours.
```
We can lean on the variable list on L23 if users want to get deeper into the technicalities of this.

---

done.